components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -51,6 +54,26 @@
 static NSString *const kAccessibilityIdentifierKeyPath = @"accessibilityIdentifier";
 static NSString *const kAccessibilityTraitsKeyPath = @"accessibilityTraits";
 
+/** The internal layout styles of the tab bar. */
+typedef NS_ENUM(NSUInteger, MDCTabBarViewInternalLayoutStyle) {
```

Comment:
How do you feel about dropping the work `internal` from the enum name? My thoughts are its in a `.m` and not exposed so as a developer I should know it's internal, I could be swayed otherwise though, I'd love to hear your thoughts.

Reply:
I intended to mark this as intentionally not usable externally. At some point we might have a set of external layout concepts, but they should be separate from these. If you prefer to drop `internal` and then rename it later to avoid naming conflicts, I can do that too.  Since this isn't a "Tip:" I'll leave it up to you.